apps/web/src/components/FormList.tsx

```diff
@@ -114,16 +115,38 @@ export const FormList = ({
               </Text>
             </GridItem>
           </Grid>
-          {sortedFormInstances.map(
-            (formInstance: FormInstanceEntity, index: number) => (
+          {sortedFormInstances
```

Comment:
Can we move this logic to the useEffect + instead of using name contains/includes statements we can use the prewritten levenshtein distance logic and take the first 5 forms with the least levenshtein distance from the search query.

Another idea is that levenshtein distance takes into consideration the whole string with considering distance, and a user typically only wants to type a few words into the search query. We could potentially limit the levenshtein search to only the first 10 characters; i.e comparing searchQuery.toLowerCase().slice(0,10) and forminstance.name.toLowerCase().slice(0,10). Through some limited testing on my end this seems to work better.